lab4/provided/direct_mapped.cc

```diff
@@ -80,13 +80,9 @@ DirectMappedCache::receiveRequest(uint64_t address, int size,
             // If the line is dirty, then we need to evict it.
             uint8_t* line = dataArray.getLine(index);
             sendMemRequest(address, memory.getLineSize(), line, -1);
-            // No response for writes, no need for valid request_id
```

Comment:
We lost this comment. It probably should have been above the sendMemRequest...

With this change, I'll merge the PR.

Thanks!